extension/framework/runner/environment.go

```diff
@@ -73,7 +74,7 @@ func (env *Environment) InitializeEnvironment() error {
 	if err != nil {
 		return fmt.Errorf("Failed to connect to database: %s", err.Error())
 	}
-	env.Environment = gohan_otto.NewEnvironment(env.dbConnection, &middleware.FakeIdentity{})
+	env.Environment = gohan_otto.NewEnvironment(env.dbConnection, &middleware.FakeIdentity{}, time.Duration(30)*time.Second)
```

Comment:
AFAIK you don't need to wrap `30` in `time.Duration()`, simple `30 * time.Second` will do the cast.